CHANGELOG.rst

```diff
@@ -2,6 +2,191 @@
 Release Notes
 =============
 
+.. _Release Notes_7.42.0:
+
+7.42.0 / 6.42.0
+======
+
+.. _Release Notes_7.42.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2023-01-19
+
+- Please refer to the `7.42.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7420>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.42.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- Downloading and installing official checks with `agent integration install`
+  is no longer supported for Agent installations that do not include an embedded
+  python3.
+
+- Starting with Agent 7.40, the Agent will fail to start when it is unable to determine the hostname, instead of silently using an irrelevant hostname (usually, a container ID).
+  Hostname resolution is key to many features and failure to determine hostname means that the Agent is not configured properly.
+  This change mostly affects Agents running in containerized environments as we cannot rely on the OS hostname.
+
+
+.. _Release Notes_7.42.0_New Features:
+
+New Features
+------------
+
+- Adding the `kube_api_version` tag to all orchestrator resources.
+
+- Kubernetes Pod events generated by the `kubernetes_apiserver` can now
+  benefit from the new `cluster-tagger` component in the Cluster-Agent.
+
+- APM OTLP: Added compatibility for the OpenTelemetry Collector's datadogprocessor to the OTLP Ingest.
+
+- The CWS agent now supports rules on mount events.
+
+- Adding a configuration option, ``exclude_ec2_tags``, to exclude EC2 instance tags from being converted into host
+  tags.
+
+- Adds detection for a process being executed directly from memory without the binary present on disk.
+
+- Introducing agent sampling rates remote configuration.
+
+- Expose process parent in SECL
+
+- Adds support for ``secret_backend_command_sha256`` SHA for the ``secret_backend_command`` executable. If ``secret_backend_command_sha256`` is used,
```

Comment:
@xornivore This PR was reverted right?

Reply:
Yup, correct, @kacper-murzyn should we remove it?

Reply:
Thank you @clarkb7 ! 🙌

Reply:
In this case yes, I can do that. Thanks for checking